tsconfig.json

```diff
@@ -18,9 +18,10 @@
     "resolveJsonModule": true,
```

Comment:
dodaj w tym pliku baseurl: "src" to będzie można importować pliki bez ../../

Reply:
fixed